shared/js/inject/mozilla.js

```diff
@@ -0,0 +1,11 @@
+SCRIPT_TO_REPLACE
```

Comment:
@sammacbeth  I couldn't think of a cleaner way to inject in the stringified code into both files. Ideall this also would be processed by rollup which would allow us to include helpers here too.

Reply:
Can this not be an import statement, with a preprocessor set up to which between chrome/firefox before rollup parses it?

Reply:
@sammacbeth not for the chrome "SCRIPT_TO_REPLACE" it's a stringified import of the code. I'm not a huge fan of stringifying code like: `functionName.toString()` as I've seen it have different results per browser.

Perhaps I could check this further if that's still an issue.